content/en/docs/reference/glossary/cloud-controller-manager.md

```diff
@@ -2,18 +2,23 @@
 title: Cloud Controller Manager
 id: cloud-controller-manager
 date: 2018-04-12
-full_link: /docs/tasks/administer-cluster/running-cloud-controller/
+full_link: /docs/concepts/architecture/cloud-controller/
 short_description: >
-  Cloud Controller Manager is a Kubernetes component that embeds cloud-specific control logic.
-
+  Control-plane component that integrates Kubernetes with third-party cloud providers.
 aka: 
 tags:
 - core-object
 - architecture
 - operation
 ---
- Cloud Controller Manager is a Kubernetes component that embeds cloud-specific control logic.
+ A Kubernetes {{< glossary_tooltip text="control-plane" term_id="control-plane" >}} component
+that embeds cloud-specific control logic. The cloud controller manager lets you run a cluster with a
+control plane that interacts with your cloud platform, and separates out the components that interact
```

Comment:
@sftim ,
nit: The second sentence, of the main description,  is long and repeats "interact" a number of times.

Reply:
Revised